src/browser.js

```diff
@@ -3,6 +3,7 @@
 import * as uuid from 'uuid/v4'
 
 export const isMobile = /Mobile/.test(navigator.userAgent)
+export const isAndroid = navigator.platform === 'Linux armv7l'
```

Comment:
Wouldn't it just be `Android`?

Or, if a particular browser needs to be excluded, some more specific Android user agent detection: 
https://stackoverflow.com/a/19915132/480608

Reply:
> Thanks! Are you able to test in a couple different mobile browsers on Android? Do we need to apply these styles to all Android browsers or just some?

This is a great question. However. our <code>navigator.platform</code> it's expecting one of these values. **Please see.**


![navigatorplatform](https://user-images.githubusercontent.com/41552663/69907003-29329800-13a3-11ea-9e47-40ec3f4c109e.png)

Raine. However. We may use this one <code>/Android/.test(navigator.userAgent)</code> like the one we have with the <code>isMobile</code> but we are not stating our whole code to be executed for the Android Platform itself, instead this one it is solely depending on the different web browser user-agent detections (Chrome, Firefox, Opera, any others).

We this actual code we are stating that we want our platform to detect Android as our main device and therefore render the elements/properties we want with the different widths for different devices as a long as we state that in our code. And for responsiveness we are using the @media CSS statement. However we may need to add more for more devices. But the bullets like so are rendering as expected



> Wouldn't it just be `Android`?
> 
> Or, if a particular browser needs to be excluded, some more specific Android user agent detection:
> https://stackoverflow.com/a/19915132/480608

<code>navigator.platform</code> it's expecting **Linux i686** or **Linux armv7l** for Android. Most New Android Devices are armv7l. Therefore <code>Android</code> won't work for us as a value.

Reply:
Raine. I must also tell you that the Browser or Platform that are not state are instead using the <code>isMobile</code> properties. Which is the default design and I know it work wells on your testing platform. But what I believe it's you need to consider revamping this state to any other like iOS instead I mean like specificying the actual device and therefore not be browser string dependant <code> navigator.userAgent </code>. But this is something I've been looking in the whole app. This is only for you to take in consideration and not an actual change I want to make neither from the bounty scope.

Reply:
Thanks so much for the clarification. I was unaware of `navigator.platform` so that makes total sense now.

Yes, I agree that the platform/user agent detection needs a lot of work to become truly cross-platform. That will be another bounty :).